homeassistant/components/zwave_me/light.py

```diff
@@ -0,0 +1,94 @@
+"""Representation of an RGB light."""
+from __future__ import annotations
+
+from typing import Any
+
+from zwave_me_ws import ZWaveMeData
+
+from homeassistant.components.light import ATTR_RGB_COLOR, COLOR_MODE_RGB, LightEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import ZWaveMeController, ZWaveMeEntity
+from .const import DOMAIN
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the rgb platform."""
+
+    @callback
+    def add_new_device(new_device: ZWaveMeData) -> None:
+        """Add a new device."""
+        controller = hass.data[DOMAIN][config_entry.entry_id]
+        rgb = ZWaveMeRGB(controller, new_device)
+
+        async_add_entities(
+            [
+                rgb,
+            ]
+        )
+
+    async_dispatcher_connect(
+        hass, "ZWAVE_ME_NEW_" + "switchRGBW".upper(), add_new_device
+    )
+    async_dispatcher_connect(
+        hass, "ZWAVE_ME_NEW_" + "switchRGB".upper(), add_new_device
+    )
+
+
+class ZWaveMeRGB(ZWaveMeEntity, LightEntity):
+    """Representation of a ZWaveMe light."""
+
+    def __init__(
+        self,
+        controller: ZWaveMeController,
+        device: ZWaveMeData,
+    ) -> None:
+        """Initialize the device."""
+        super().__init__(self, controller=controller, device=device)
+
+    def turn_off(self, **kwargs: Any) -> None:
+        """Turn the device on."""
+        self.controller.zwave_api.send_command(self.device.id, "off")
+
+    def turn_on(self, **kwargs: Any):
+        """Turn the device on."""
+        color = kwargs.get(ATTR_RGB_COLOR)
+
+        if color is None:
+            color = [122, 122, 122]
```

Comment:
`color` above is an optional three item integer tuple. We should keep the type here.
```suggestion
            color = (122, 122, 122)
```